### PR TITLE
add recipe to build ctaplot package

### PR DESCRIPTION
**Pull request by vuillaut**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by vuillaut** [comment]
@conda-forge-admin, please ping conda-forge/help-python
Hi @conda-forge/help-python
Windows build is failing. I would be fine excluding windows from the supported platforms, is there a way to do that please?

File: recipes/ctaplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/ctaplot/meta.yaml:1]: Ping!

File: recipes/ctaplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 907a734af1c535148f3efde24bf56ba28ffee7e9ba7f4c171b38ed8396b81c21
+
+build:
+  noarch: python
```
Comment (ocefpaf) [inline recipes/ctaplot/meta.yaml:20]: Cannot be noarch and depend on a compiler like you are doing below.  Try to use grayskull to generate this recipe, it usually creates one that is almost merge ready.

File: recipes/ctaplot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
```
Comment (ocefpaf) [inline recipes/ctaplot/meta.yaml:28]: Are you sure this package has compiled extensions?

File: recipes/ctaplot/meta.yaml (hunk 2 above)
Comment (vuillaut) [reply recipes/ctaplot/meta.yaml:15]: Thanks!

**Comment by vuillaut** [comment]
Thank you, I have updated the recipe.

**Pull request merged**